ovirt-imageio.spec.in

```diff
@@ -94,8 +132,9 @@ Daemon providing image transfer service on oVirt hosts.
 
 %files daemon
 %{python3_sitearch}/%{srcname}/admin
-%{_bindir}/%{name}
-%{_bindir}/%{name}ctl
+# Marking below
+%attr(755, root, root) %{_bindir}/%{name}
+%attr(755, root, root) %{_bindir}/%{name}ctl
```

Comment:
Why is this needed? Isn't this the default?

Reply:
I guess because of those ">" above. Indeed we should just sed -i instead

Reply:
Not needed anymore, Marcin's solution is better